app/Domain/Subscriptions/Models/SubscriptionModel.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Subscriptions\Models;
+
+use App\Models\UuidModel;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+final class SubscriptionModel extends UuidModel
+{
+    use SoftDeletes;
+
+    protected $table = 'subscriptions';
```

Comment:
While reviewing #4 I realized that because we store the subscriptions in a table, we won't have any subscriptions locally. And we'll also have to keep the subscriptions on 3 environments in sync: acc, test and prod. (In theory it doesn't matter what the subscriptions are on acc/test, but for predictability it might be best to keep them the same as on prod.)

Maybe we should keep them in a config file instead so we can easily keep them in sync on all 4 environments? (local, acc, test, prod)

Reply:
Or we could create a [seeder](https://laravel.com/docs/9.x/seeding) for that, which could become part of the deployment phase like the migrations.
I will create a seeder for now and we can finetune later when things are more concrete.

Reply:
Can we avoid the subscriptions being editable in Nova with a seeder? And won't a seeder be a bit complex for someone like @corneelwille to edit the subscription info when needed?

Reply:
> Can we avoid the subscriptions being editable in Nova with a seeder?
We can make the subscriptions read-only in Nova

> And won't a seeder be a bit complex for someone like @corneelwille to edit the subscription info when needed?
Yes, but the seeder can be filled with values from a config.

I think this is something we can implement in a follow-up PR.

Reply:
Okay sounds good. I only don't see why we need a database table, Eloquent model, and database seeder for subscriptions if they will be read-only and read from a config file in the end. 😄